docs/1.getting-started/9.layers.md

```diff
@@ -47,8 +47,8 @@ Nuxt uses [unjs/c12](https://c12.unjs.io) and [unjs/giget](https://giget.unjs.io
 Read more about layers in the **Layer Author Guide**.
 ::
 
-::tip{icon="i-ph-video-duotone" to="https://www.youtube.com/watch?v=lnFCM7c9f7I" target="_blank"}
-Watch Learn Vue video about Nuxt Layers.
+::tip{icon="i-ph-video-duotone" to="https://www.youtube.com/watch?v=fr5yo3aVkfA&t=271s" target="_blank"}
```

Comment:
Maybe it would be worth linking to both videos?

Reply:
Sure I will add both videos as two tips in docs